inst/assets/BS4/pkgdown.css

```diff
@@ -465,3 +465,35 @@ summary {
 details p {
   margin-top: -.5rem;
 }
+
+/* tabsets */
+.nav-row {
+  flex-direction: row;
+}
+
+.tab-content {
+  padding: 1rem;
+}
+
+.tabset-pills .tab-content {
+  border-left: solid 1px #e5e5e5;
+  border-bottom: solid 1px #e5e5e5;
+}
+
+/* https://observablehq.com/@rkaravia/css-trick-tabs-with-consistent-height */
```

Comment:
This is to make the height of tabs consistent, I found the previous behaviour annoying as clicking on a tab would make content appear (good) but if it was smaller than the previous tab, it'd make the next sections after the tabset jump up (bad IMO).